Kitodo/src/main/java/org/kitodo/production/migration/XmlGenerator.java

```diff
@@ -50,21 +51,22 @@ static String generateTask(Task task) {
      *            for sequence flow
      * @return generated task
      */
-    static String generateTask(Task task, String sourceReference, String targetReference) {
+    static String generateTask(Task task, String sourceReference, String targetReference, int ordering) {
```

Comment:
You could probably remove the third parameter `targetReference` - because it is always `"Task_" + order` - so simplify the method signature a little bit to
`static String generateTask(Task task, String sourceReference, int ordering)`